test/test_wavefront_python_sdk.py

```diff
@@ -0,0 +1,210 @@
+import unittest
+from uuid import UUID
+
+from wavefront_python_sdk.common.utils import sanitize, \
+    metric_to_line_data, histogram_to_line_data, tracing_span_to_line_data
+from wavefront_python_sdk.entities.histogram import HistogramGranularity
+
+
+class TestUtils(unittest.TestCase):
+
+    def test_sanitize(self):
+        """
+        Test wavefront_python_sdk.common.utils.sanitize()
+        """
+        self.assertEqual("\"hello\"", sanitize("hello"))
+        self.assertEqual("\"hello-world\"", sanitize("hello world"))
+        self.assertEqual("\"hello.world\"", sanitize("hello.world"))
+        self.assertEqual("\"hello\\\"world\\\"\"", sanitize("hello\"world\""))
+        self.assertEqual("\"hello'world\"", sanitize("hello'world"))
+
+    def test_metric_to_line_data(self):
+        """
+        Test wavefront_python_sdk.common.utils.metric_to_line_data()
+        """
+        self.assertEqual(
+            "\"new-york.power.usage\" 42422.0 1493773500 source=\"localhost\"" +
```

Comment:
Python has implicit string concatenation inside parens, so ` + ` is not needed here and elsewhere.